WIP-CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Configuration for the certificate issuers has been changed and requires running the [migration script](migration/v0.6.x-v0.7.x/migrate-issuer-config.sh).
 - Configuration for harbor has been changed and requires running init and apply again.
+- Helm chart for grafana has been changed and requires running apply again.
```

Comment:
What does this mean? To me an upgrade of Compliant Kubernetes will in itself mean running apply. Does this mean after/before that upgrade you need to run apply for grafana an extra time? If it is just the normal apply then I don't think it is worth mentioning.

Reply:
Could have been after my request, must have been thinking of apply as an init step :confused: Sorry @JonasKop !

Reply:
Haha no worries @danielharr, I fixed that now!